server/aesdsocket.c

```diff
@@ -226,7 +215,7 @@ void* ReceiveAndSendPackets(void* thread_data) {
                     if (send_status == -1) {
                         perror("send");
                         syslog(LOG_ERR, "send");
-                        close(t->outfile_fd);
+                        close(outfile_fd);
```

Comment:
@vido2373 rather than duplicating this cleanup logic everywhere, think about either using a design with common return path or a function.